src/settings.ts

```diff
@@ -36,6 +36,6 @@ export const DEFAULT_SETTINGS: Settings = {
   hoverDelay: 250,
   numberResetSelector: '',
   scope: '',
-  contentTemplate: `<aside class="littlefoot-footnote" id="fncontent:<% id %>" alt="Footnote <% number %>" aria-live="polite"><div class="${CLASS_WRAPPER}"><div class="${CLASS_CONTENT}" tabindex="0"><% content %></div></div><div class="${CLASS_TOOLTIP}"></div></aside>`,
```

Comment:
Looks good, could you update the README as well, please?

The change also breaks a few tests if you could have a look at those.

Reply:
I updated the README. Regarding the changes in the templated, I had to remove also aria-describedby, for the same reason as aria-controls, i.e. most of the time the referenced element is not in the dom.